modules/nvidia-gpu-prerequisites.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * architecture/nvidia-gpu-architecture-overview.adoc
+
+
+:_content-type: CONCEPT
+[id="nvidia-gpu-prerequisites_{context}"]
+= NVIDIA GPU prerequisites
+
+* A working OpenShift cluster up and running with a GPU worker node.
```

Comment:
"at least one GPU worker node" would be more accurate, IMO

Reply:
Changed.